upgrade/capsule.py

```diff
@@ -56,10 +55,6 @@ def satellite_capsule_setup(satellite_host, capsule_hosts, os_version,
             non_responsive_host.append(cap_host)
         else:
             execute(host_ssh_availability_check, cap_host)
-        # Update the template once 1829115 gets fixed.
-        execute(workaround_1829115, host=cap_host)
-        if not bz_bug_is_open(1829115):
-            logger.warn("Please update the capsule template for fixed capsule version")
```

Comment:
@Lukas Pramuk If you removed this workaround, then the satellite-capsule integration will get failed, the workaround placed at capsule side, you have to update the template(remove ) before removing this workaround.

Reply:
I will handle it if needed

Reply:
I've worked around the issue differently - before capsule templatization I've run these commands:
```
mkdir -p /root/.hammer/cli.modules.d/
touch /root/.hammer/cli.modules.d/dummy.yml
```
No need for baked-in `/usr/share/katello/hostname-change.rb` brought from 6.10.0 and causing other issues with upgrades - error 78 - hotfix detected